### PR TITLE
Util: Update `plextrac logs` command

### PR DESCRIPTION
**Pull request by Michael-Burke**
- Updated logs command to filter by service
- Updated `plextrac help` command and to include logs command

## Testing:
Help Command:
image

Logs Command output:
image

`plextrac logs` without a flag and service will show all logs just like before

### CODE REVIEW

**Review by mmorgan-pt — DISMISSED** [review]
very nice feature add! I like the extra colors too!

**Comment by Michael-Burke** [comment]
Updated the help to show more specific services for those unfamiliar with docker-compose services

File: src/plextrac [hunk 1]
```diff
@@ -159,6 +160,11 @@ function main() {
         mod=mod_dist
         break
         ;;
+      "-s")
+        LOG_SERVICE=${2-''}
+        shift
+        shift
+        ;;
```
[the remark this answers is not in the document]
Comment (Michael-Burke) [reply src/plextrac]: Added `--service`  into help and actual command

**Review by jesseops — APPROVED** [review]
I like it, great QoL update.

**Pull request merged**